### DISCUSSION TITLE
Heroku Support.

### DISCUSSION THREAD

**Discussion by pbkx** [titaniumnetwork-dev/Vanadium#14] [Q&A] [answered]
I've left developement on Node.JS for a while for Linux and have just gotten back into Proxy developement.

Was just wondering why Heroku support was dropped.

**Comment by Null-n-V0id** [comment, marked as answer]
I dropped Heroku support since Heroku now actively blocks any open proxy service running on their network. Glitch.com has been shown to be a promising alternative, with their support for custom domains and their tolerance of proxies.
